app/controllers/beta_releases_controller.rb

```diff
@@ -0,0 +1,32 @@
+class BetaReleasesController < SignedInApplicationController
+  include Tabbable
+  before_action :require_write_access!
+  before_action :set_release_platform_run, only: %i[create]
+
+  def index
+    live_release!
+    @app = @release.app
+  end
+
+  def create
```

Comment:
ensure the release platform run is active. either here or in Coordinator action.